addons/field_rations/XEH_postInit.sqf

```diff
@@ -19,7 +19,17 @@ if !(hasInterface) exitWith {};
     GVAR(hudInteractionHover) = false;
     GVAR(hudIsShowing) = false;
 
-    [] call FUNC(addRefillActions);
+    // Add refill water actions to water sources from config
+    private _filter = QUOTE(getNumber (_x >> 'scope') == 2 && {getNumber (_x >> QQGVAR(refillSource)) > 0});
+    private _waterSources = _filter configClasses (configFile >> "CfgVehicles");
```

Comment:
you technically could go the ACE arsenal route and prep this at preStart to spare the players some loading screen time.

Reply:
done